src/client/datascience/editor-integration/cellhashprovider.ts

```diff
@@ -106,6 +107,10 @@ export class CellHashProvider implements ICellHashProvider {
     }
 
     public async addCellHash(cell: NotebookCell) {
+        // Skip markdown cells as they are never actually executed
```

Comment:
Should this skip raw cells too? As in make it only work for 'code' cells? I'm not entirely sure if we allow raw execution so maybe not.

Reply:
We don't support raw cells in the interactive window afaik so this should be fine.